static/scripts/courses.js

```diff
@@ -125,6 +125,11 @@ $(document).ready(function () {
                 $shareModal.appendTo('body').modal('show');
 
                 $("label[for='shareToken']").text('Verteile folgenden Code an einen Lehrer-Kollegen, um den Kurs mit diesem zu teilen. Die Funktion befindet sich auf der Übersichtsseite für Kurse.');
+                let image = kjua({text: window.location.origin + '/courses?import=' + data.shareToken, render: 'image'});
```

Comment:
Do not use window.location.origin !!! https://stackoverflow.com/questions/22564167/window-location-origin-gives-wrong-value-when-using-ie instead, think about creating the text property inside an hbs file using the client controller

Reply:
I also thought about this, but I don't think theres a good solution for implementing this in the hbs files. The Problem is, that we do not have a full page reload after creating a course. We create a course via ajax and therefore can't populate the values we need directly inside the hbs file.
The only "ok"-solution I could think of is to store some data in hidden input fields (like the base url) and read these values here, instead of using window.location.origin.

Reply:
ok, so we should use the _The only "ok"-solution I could think of is to store some data in hidden input fields (like the base url) and read these values here, instead of using window.location.origin.


_

Reply:
Ok, will do.